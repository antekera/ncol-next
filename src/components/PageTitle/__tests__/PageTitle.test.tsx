import { render } from '@testing-library/react'
import { PageTitle } from '..'

describe('PageTitle', () => {
  test('should match snapshots', () => {
    const { container } = render(<PageTitle text='Post title' />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
