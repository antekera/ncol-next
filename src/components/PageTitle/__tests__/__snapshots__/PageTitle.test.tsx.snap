// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle should match snapshots 1`] = `
<div
  class="page-title"
>
  <div
    class="container text-left"
  >
    <h1
      class="page-title-heading"
    >
      <span>
        Post title
      </span>
    </h1>
  </div>
</div>
`;
